components/table/table-col-sort-button.js

```diff
@@ -129,6 +166,7 @@ export class TableColSortButton extends LocalizeCoreElement(RtlMixin(FocusMixin(
 		super();
 		this.dataType = 'words';
 		this.desc = false;
+		this._hasDropdownItems = false;
```

Comment:
We generally set internal properties alphabetically below the external ones (i.e., it would go after `this.nosort`)